libs/net/template/Models/Reports/ReportEngineContentModel.cs

```diff
@@ -44,25 +44,35 @@ public ReportEngineContentModel() : base(Array.Empty<ContentModel>())
     /// </summary>
     /// <param name="report"></param>
     /// <param name="content"></param>
-    public ReportEngineContentModel(API.Areas.Services.Models.Report.ReportModel report, IEnumerable<ContentModel> content)
+    /// <param name="options"></param>
+    public ReportEngineContentModel(API.Areas.Services.Models.Report.ReportModel report, IEnumerable<ContentModel> content, Config.TemplateOptions options)
         : base(content)
     {
         this.Settings = report.Settings;
         this.OwnerId = report.OwnerId;
+        this.SubscriberAppUrl = options.SubscriberAppUrl;
```

Comment:
Fixed an issue where the template was not getting this information for the report extension methods